models/decision.go

```diff
@@ -79,6 +79,12 @@ type ScenarioExecution struct {
 	OrganizationId      string
 }
 
+type RuleExecutionStat struct {
```

Comment:
Here, I think we also need to include, at least, the rule id (or perhaps more conveniently the stable id, that will be introduced)

Reply:
(and in the dto, of course)